app/src/renderer/vuex/modules/delegation.js

```diff
@@ -165,7 +166,10 @@ export default ({ node }) => {
       })
 
       // optimistic update the atoms of the user before we get the new values from chain
-      commit(`setAtoms`, user.atoms - amount)
+      commit(`updateWalletBalance`, {
+        denom,
+        amount: liquidAtoms - amount
+      })
       // optimistically update the committed delegations
       Vue.set(
```

Comment:
Minor: it's not part of this PR but when I see an action modifying the state I think we should add a comment TODO or fix it and create a mutation to do that.